project/deps.sc

```diff
@@ -113,3 +113,13 @@ object Docker {
   def alpineTestImage =
     "alpine@sha256:234cb88d3020898631af0ccbbcca9a66ae7306ecd30c9720690858c1b007d2a0"
 }
+
+def customRepositories =
+  Seq(
+    coursier.Repositories.sonatype("snapshots")
+    // Uncomment for local development
+    // coursier.MavenRepository(
+    //   (os.Path(System.getProperty("user.home")) / ".m2" / "repository")
+    //     .toNIO.toUri.toASCIIString
+    // )
```

Comment:
IIRC you can add MavenLocal repo with: `coursier.LocalRepositories.Dangerous.maven2Local`